operator/operator.make

```diff
@@ -46,6 +46,11 @@ MANIFESTS_DIR=$(REPO_ROOT)/deploy/kustomize/olm-catalog
 CATALOG_DIR=$(REPO_ROOT)/deploy/olm-catalog
 BUNDLE_DIR=$(REPO_ROOT)/deploy/olm-bundle/$(MAJOR_MINOR_PATCH_VERSION)
 CRD_DIR=$(REPO_ROOT)/deploy/crd
+# CHANNELS: A comma-separated list of channels the generated
+# bundle belongs to.
+CHANNELS?=stable
```

Comment:
Should this include `alpha`?

Please include instructions when to use more than one channel or link to further information.

Reply:
> Should this include `alpha`?

Maybe yes. I myself not sure from where the OLM gets the channels in the bundle format, because these are defined in annotations inside the version-specific folders. There is no common file to define all available channels, where it used to be in package format. Anyways I will add the 'alpha' to the channel list.

Reply:
Please add it and I'll merge.

Reply:
Done.